# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoaderOwner.h
# Commit: 95b350572bbf
# Full Hash: 95b350572bbf079fdf6613b5a721c6ee5023fc6e
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1646563
# File Overlap Count: 1
# Description:
#   Bug 1646563 - Remove BrowserId from nsFrameLoaderOwner, r=nika
#   
#   We can just use BrowsingContext::BrowserId directly, so it's unnecessary to have
#   the field on nsFrameLoaderOwner as well.
#   
# ==============================================================================

diff -r 4eab38aba3fe -r 95b350572bbf dom/base/nsFrameLoaderOwner.h
--- a/dom/base/nsFrameLoaderOwner.h	Mon Jun 22 21:30:31 2020 +0000
+++ b/dom/base/nsFrameLoaderOwner.h	Mon Jun 22 21:46:03 2020 +0000
@@ -74,12 +74,6 @@
 
   void SubframeCrashed();
 
-  // Prepare for a frame to be removed from its current DOM tree.
-  void UnbindFromTree();
-
-  uint64_t GetBrowserId() { return mBrowserId; }
-  void SetBrowserId(uint64_t aBrowserId) { mBrowserId = aBrowserId; }
-
  private:
   bool UseRemoteSubframes();
 
@@ -101,8 +95,6 @@
                               std::function<void()>& aFrameLoaderInit,
                               mozilla::ErrorResult& aRv);
 
-  uint64_t mBrowserId = 0;
-
  protected:
   virtual ~nsFrameLoaderOwner() = default;
   RefPtr<nsFrameLoader> mFrameLoader;